pom.xml

```diff
@@ -1605,41 +1606,49 @@
     <!-- Profiles -->
     <profiles>
         <!--
-          Name: modularizedJdk
-          Descr: various workarounds activation for modularized JDK
+          Name: JDK9
+          Descr: various workarounds activation for JDK9
         -->
         <profile>
-            <id>modularizedJdk</id>
+            <id>JDK9</id>
             <activation>
                 <jdk>9</jdk>
             </activation>
             <properties>
-                <!-- [WFCORE-1431] remove SASL workaround -->
-                <modular.jdk.args>--add-exports=java.security.sasl/com.sun.security.sasl.digest=ALL-UNNAMED
-                    --add-exports=java.security.sasl/com.sun.security.sasl=ALL-UNNAMED
-                    --add-exports=jdk.unsupported/sun.reflect=ALL-UNNAMED
-                    --add-exports=jdk.unsupported/sun.misc=ALL-UNNAMED
+                <modular.jdk.args>
```

Comment:
This change looks good but we need to keep:
--add-exports=java.base/sun.nio.ch=ALL-UNNAMED